client/src/components/MapGeocoderWrapper.vue

```diff
@@ -29,20 +30,27 @@ import mapboxgl from 'mapbox-gl';
  */
 export default defineComponent({
   name: 'MapGeocoderWrapper',
-  setup() {
+  setup(props, { emit }) {
+    const visible = computed({
```

Comment:
What does this function call do?

Reply:
Added a comment but essentially binds the prop value of isExpanded to the v-model value expandSearch. V-model is just a two way binding. I put it in the template instead which may make it more straight forward

Reply:
Sounds good, thanks.